conda/plugins/hookspec.py

```diff
@@ -256,6 +257,45 @@ def conda_health_checks():
         """
         yield from ()
 
+    @_hookspec
+    def conda_post_transactions(self) -> Iterable[CondaPostTransaction]:
+        """Register post-transaction hooks.
+
+        Post-transaction hooks run after all other actions run in a
+        UnlinkLinkTransaction. For information about FinalTransactionAction,
+        see :class:`~conda.core.path_actions.FinalTransactionAction`.
+
+        **Example:**
+
+        .. code-block:: python
+
+            from conda import plugins
+            from conda.core.path_actions import FinalTransactionAction
+
+
+            class PrintAction(FinalTransactionAction):
+                def execute(self):
+                    print(
+                        self.transaction_context,
+                        self.target_prefix,
+                        self.unlink_precs,
+                        self.link_precs,
+                        self.remove_specs,
+                        self.update_specs,
+                        self.neutered_specs,
+                    )
```

Comment:
```suggestion
                    )
                def verify(self):
                    print("Performing verify actions...")
                    self._verified = True

                def reverse(self):
                    print("Reversing...")

                def cleanup(self):
                    print("Cleaning up...")
```

It would be nice for the example to define all abstract methods.